<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>231</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Lantern</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QComboBox" name="cb_ip">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>192</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>192</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="editable">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>emb2.extcam.xyz</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="le_port">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>60</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>64</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>64</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>9991</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_connect">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>96</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>96</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>Connect</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QTabWidget" name="tab_main">
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Lantern</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <item>
                 <widget class="QLabel" name="lab_status">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>32</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>32</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>state:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_state_text">
                  <property name="minimumSize">
                   <size>
                    <width>92</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>128</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>UNKNOWN</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="lab_color">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>32</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>32</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>color:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_color_text">
                  <property name="minimumSize">
                   <size>
                    <width>92</width>
                    <height>0</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>128</width>
                    <height>16777215</height>
                   </size>
                  </property>
                  <property name="text">
                   <string>UNKNOWN</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="le_color">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                  <property name="maximumSize">
                   <size>
                    <width>20</width>
                    <height>20</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="horizontalSpacer_4">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>40</width>
                    <height>20</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QRadioButton" name="tb_online">
              <property name="text">
               <string>on air</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QRadioButton" name="rb_offline">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="text">
               <string>on time</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTimeEdit" name="te_offline">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="time">
               <time>
                <hour>12</hour>
                <minute>0</minute>
                <second>0</second>
               </time>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Log</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextEdit" name="le_log">
            <property name="lineWrapMode">
             <enum>QTextEdit::NoWrap</enum>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>420</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
